### DISCUSSION TITLE
Adding bookmark for Saved Filter

### DISCUSSION THREAD

**Discussion by markkuleinio** [netbox-community/netbox#14590] [Q&A] [answered]
Let me start by saying that I haven't used saved filters much yet.

But, is there a way to add a bookmark for a saved filter? There is no Bookmark button in the saved filter page.

**Comment by jeremystretch** [comment, marked as answer]
Bookmarks currently work only for object detail views (not list views). What you could do instead is create a note widget with a link to the desired list view with the saved filter applied, although that would appear only on the dashboard.

**Reply by markkuleinio** [reply]
Right, I see there is already a feature request #13694 related to this, pending closure
